lib/generate-sitemap.js

```diff
@@ -16,18 +16,17 @@ function generateSitemap(batfishConfig) {
     path.join(batfishConfig.outputDirectory, 'sitemap.xml')
   );
   return new Promise((resolve, reject) => {
-    sitemapStatic(
-      sitemapWriter,
-      {
+    try {
```

Comment:
I think that try-catch is not going to work here, either. On closer inspection, I think we'll have to rely on `sitemapWriter` emitting its stream events. We should add `sitemapWriter.on('error', reject)` and `sitemapWriter.on('finish', () => resolve())`, I think.